src/shared/components/TextField/TextField.stories.tsx

```diff
@@ -17,6 +19,17 @@ const TemplateWithControlledInput: Story<TextFieldProps> = (args) => {
   return <TextField {...args} onChange={(e) => setValue(e.target.value)} value={value} />
 }
 
+const TemplateWithUncontrolledInput: Story<TextFieldProps> = (args) => {
```

Comment:
In uncontrolled component, I'm getting this, but it only happens in stories. I couldn't figured why.
![image](https://user-images.githubusercontent.com/51168865/107215457-a035d880-6a0b-11eb-887b-32700a9400f3.png)
I tested this component in `HomeView` and everything works fine.

Reply:
Seems it's some problem with Storybook - https://github.com/storybookjs/storybook/issues/6471 Let's not worry about it